apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-modal/index.tsx

```diff
@@ -24,24 +24,20 @@ interface Props {
 }
 
 const LaunchModal: React.FunctionComponent< Props > = ( { onClose } ) => {
+	const { siteId } = React.useContext( LaunchContext );
+
 	const { step: currentStep, isSidebarFullscreen } = useSelect( ( select ) =>
 		select( LAUNCH_STORE ).getState()
 	);
-	const { launchSite } = useDispatch( LAUNCH_STORE );
-
 	const [ isLaunching, setIsLaunching ] = React.useState( false );
 
-	const { isPaidPlan } = useSite();
+	const { launchSite } = useDispatch( SITE_STORE );
```

Comment:
[Only for testing]: `launchSiteComplete` can be used here to simulate a site launch without actually launching the site.